packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -30,7 +44,16 @@ export const ChatList = (props: MgtTemplateProps & IChatListItemInteractionProps
     <FluentThemeProvider fluentTheme={FluentTheme}>
       <FluentProvider theme={webLightTheme}>
         {chats.map(c => (
-          <ChatListItem key={c.id} chat={c} myId={chatState.userId} onSelected={props.onSelected} />
+          <ChatListItem
+            key={c.id}
+            chat={c}
+            myId={chatState.userId}
+            onSelected={props.onSelected}
+            isSelected={c.id === selectedItem}
+            isRead={readItems.includes(c.id ?? '')}
+            onChatItemSelected={updateSelectedChatListItem}
```

Comment:
Why would we have onSelected and onChatItemSelected? Intuitively I have no idea what to make of that.

Reply:
I update the naming. Functionality essentially remains the same though.